datapackage.yaml

```diff
@@ -0,0 +1,30 @@
+name: s-and-p-500-companies
+title: S&P 500 Companies with Financial Information
+version: "2.0"
+licenses:
+  - name: ODC-PDDL-1.0
+    path: http://opendatacommons.org/licenses/pddl/
+    title: Open Data Commons Public Domain Dedication and License v1.0
+resources:
+  - name: constituents
+    path: data/constituents.csv
+    format: csv
+    mediatype: text/csv
+    schema:
+      fields:
+        - name: Symbol
+          type: string
+        - name: Security
+          type: string
+        - name: GICS Sector
+          type: string
+        - name: GICS Sub-Industry
+          type: string
+        - name: Headquarters Location
+          type: string
+        - name: Date added
+          type: string
+        - name: CIK
+          type: string
+        - name: Founded
+          type: string
```

Comment:
Moved to YAML as it makes it more readable and I is what modern Frictionless data packages are using around GitHub.